examples/redis-cluster-gke/README.md

```diff
@@ -0,0 +1,55 @@
+# Deploying Redis Cluster on GKE
+
+This is a sample K8s configuration files to deploy a Redis Cluster on GKE.
+
+## How to use
+
+1. Provision a GKE cluster.
+
+```
+$ gcloud container clusters create redis-cluster --num-nodes 6 --machine-type n1-standard-8 --image-type COS --disk-type pd-standard --disk-size 100 --enable-ip-alias --create-subnetwork name=redis-subnet
+```
+
+Please specify your own project ID or prefered zone if you asked.
```

Comment:
Perhaps you can reword that to:
If prompted, please specify your preferred project id and zone.

Reply:
Modified.